src/Dnn.vNext/Pages/_Admin.cshtml

```diff
@@ -95,16 +95,73 @@
                     url: 'api/page/EditPage',
                     type: 'POST',
                     success: function (data) {
+                        $('#persona-edit-button').hide();
+                        $('#persona-cancel-edit-button').show();
+
                         if ($('footer').length) {
                             $('footer').remove();
                         }
                         else {
-                            $('#main').append(data);    
-                        }                        
+                            $('#main').append(data);
+                        }
                     }
                 });
             });
         });
+
+        $(function () {
+            $('#persona-cancel-edit-button').click(function () {
+                $('#persona-edit-button').show();
+                $('#persona-cancel-edit-button').hide();
+                $('footer').remove();
+            });
+        });
+    </script>
+
+
+    <script type="text/javascript">
```

Comment:
We should be able to put this all in one `<script>` block

Reply:
I have updated the code and put into one code block